static/css/profile.css

```diff
@@ -42,8 +41,17 @@ main > header {
 .profile-intro section {
   flex-basis: 100%;
   order: 2;
+  margin-top: 5rem;
+}
+.profile-intro section  a {
```

Comment:
The link to trips has been improved since: https://github.com/neomad-team/neomad.org/blob/master/static/css/profile.css#L47